routes/annotations.js

```diff
@@ -98,6 +63,18 @@ exports.updateMutation = function updateMutation(req, res) {
 
 };
 
+exports.createMutationRef = function(req, res) {
```

Comment:
Maybe prepend these function names with "redirect", so it's clear this is no longer handled here.